tests/test_send_stats_to_performance_platform.py

```diff
@@ -0,0 +1,359 @@
+import json
```

Comment:
I was losing the will to live :/
I can add a test to check it makes the right call if you reckon it's worth it.  I didn't feel like there would be a lot of value in it, but I tend to be slacker than I should be about these things.